srcpkgs/NetworkManager-l2tp/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'NetworkManager-l2tp'
+pkgname=NetworkManager-l2tp
+version=1.8.2
+revision=1
+build_style=gnu-configure
+configure_args="--runstatedir=/run"
+hostmakedepends="pkg-config intltool glib-devel"
+makedepends="gtk+3-devel libglib-devel libsecret-devel network-manager-applet-devel
+libressl-devel nss-devel ppp-devel"
+depends="strongswan xl2tpd ppp"
+short_desc="NetworkManager VPN plugin for L2TP/IPsec"
+maintainer="Douglas Kosovic <doug@uq.edu.au>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/nm-l2tp/NetworkManager-l2tp"
+distfiles="https://github.com/nm-l2tp/NetworkManager-l2tp/releases/download/${version}/${pkgname}-${version}.tar.xz"
+checksum=1d80986dc88951e298446833d314bab7af0a933e736960d6b3b025e6b7faca94
+
+do_configure() {
+	if [ "$CROSS_BUILD" ]; then
+		PPP_VERSION=$(sed -n 's/^\#define\s*VERSION\s*"\([^\s]*\)"$/\1/p' /usr/${XBPS_CROSS_TRIPLET}/usr/include/pppd/patchlevel.h)
+	else
+		PPP_VERSION=$(sed -n 's/^\#define\s*VERSION\s*"\([^\s]*\)"$/\1/p' /usr/include/pppd/patchlevel.h)
```

Comment:
I'll applied this on top of your change:
```diff
diff --git a/srcpkgs/NetworkManager-l2tp/template b/srcpkgs/NetworkManager-l2tp/template
index 16bd590a97..5fc1665715 100644
--- a/srcpkgs/NetworkManager-l2tp/template
+++ b/srcpkgs/NetworkManager-l2tp/template
@@ -18,11 +18,7 @@ checksum=1d80986dc88951e298446833d314bab7af0a933e736960d6b3b025e6b7faca94
 # Automatically determine the pppd plugin dir location, but will need to
 # revbump this package whenever version of ppp package is updated.
 do_configure() {
-	if [ "$CROSS_BUILD" ]; then
-		PPP_VERSION=$(sed -n 's/^\#define\s*VERSION\s*"\([^\s]*\)"$/\1/p' /usr/${XBPS_CROSS_TRIPLET}/usr/include/pppd/patchlevel.h)
-	else
-		PPP_VERSION=$(sed -n 's/^\#define\s*VERSION\s*"\([^\s]*\)"$/\1/p' /usr/include/pppd/patchlevel.h)
-	fi
+	PPP_VERSION=$(awk '/VERSION/{print $3}' ${XBPS_CROSS_BASE}/usr/include/pppd/patchlevel.h)
 
 	./configure ${configure_args} --with-pppd-plugin-dir=/usr/lib/pppd/${PPP_VERSION}
 }
```
Thought?

Reply:
I like how `${XBPS_CROSS_BASE}`simplified things.

awk version of `PPP_VERSION` ends up being `"2.4.7"` while the sed version didn't have quotes, but `--with-pppd-plugin-dir=/usr/lib/pppd/"2.4.7"` will work fine as a configure argument, so I'm okay with it.

Reply:
> awk version of `PPP_VERSION` ends up being `"2.4.7"` while the sed version didn't have quotes, but `--with-pppd-plugin-dir=/usr/lib/pppd/"2.4.7"` will work fine as a configure argument, so I'm okay with it.

That's right. I aware that `awk 'print $3'`'s output will be quoted with d-quotes.
But, the quotes will be stripped out with shell expansion.

Reply:
Not sure if I'm the one that needs to commit the change or you'll apply the change?

Reply:
One (unimportant) questions: does the package work if `/etc/dbus-1/system.d/nm-l2tp-service.conf` is moved into `/usr/share/dbus-1/system.d`?

In any case, I can commit myself.

Reply:
The current Debian package has a patch that does that, so can confirm it should work with that location.

The next upstream version will use `/usr/share/dbus-1/system.d` by default with commit https://github.com/nm-l2tp/NetworkManager-l2tp/commit/1fb9197b628f4b010e515a49b55a2d20ed18a9cf.